### ISSUE TITLE
New version release

### ISSUE THREAD

**Issue by mtelka** [mintchaos/typogrify#61]
The latest released version of `typogrify` is 2.0.7 and it was released more than 10 years ago (September 2014).  I see some activity in git up to 2022 so it looks like the project is not completely dead.  Would you mind to release new version please?

Also, it would be great if #56 is solved.

Or, is `typogrify` deprecated?

Thank you.

**Comment by mtelka** [comment]
We do have `typogrify` [packaged for OpenIndiana](https://github.com/OpenIndiana/oi-userland/pull/10576).  Primarily because it is a dependency for `gi-docgen`.  While reviewing our Python packages for possible deprecates I simply noticed that `typogrify` is our only Python package with so long time that passed since the latest release (10 years is extremely long in the Python world).  In addition, there is the `smartypants` issue unresolved yet.

Our current `typorify` works with Python 3.9 and because I'm preparing upgrade to Python 3.13 I wanted to make sure the upgrade will be as smooth as possible.  Since I was not sure what is the current status I just raised this request.

I'm glad to hear that `typogrify` is still live.

Thank you for your input and sorry for noise.
